api/app.js

```diff
@@ -12,6 +12,12 @@ const app = express();
 
 app.use(express.json());
 
+// M1 Version
+// const corsOptions = {
+//   origin: "https://snake.adamsackfield.uk",
```

Comment:
Can delete this commented code I think? we were able to run it without this commented in